lib/dataTrigger/dpnsTriggers/createDomainDataTrigger.js

```diff
@@ -0,0 +1,45 @@
+const DataTriggerExecutionError = require('../../errors/DataTriggerExecutionError');
+const calculateHash = require('../../util/hash');
+const Document = require('../../document/Document');
+const DataTrigger = require('../DataTrigger');
+
+/**
+ * @param {Document} document
+ * @param {DataTriggerExecutionContext} context
+ * @return {Promise<boolean>}
+ */
+async function createDomainDataTrigger(document, context) {
+  const {
+    hash, normalizedLabel, label, parentDomainHash, records,
+  } = document.getData();
+
+  if (hash !== calculateHash(Buffer.from(normalizedLabel)).toString('hex')) {
+    throw new DataTriggerExecutionError(document, context, 'Document hash doesn\'t match actual hash');
```

Comment:
throw new Error('Document hash doesn\'t match actual hash')

Reply:
Well, there will be no error message then, which I guess can be a little bit confusing for the developer

Reply:
return trigger result